Automattic/jetpack
# Sharing: ensure the feature is available in offline mode

#### Changes proposed in this Pull Request:

﻿The feature can be used without a connection to a WordPress.com account.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start a brand new site
* Enable Offline mode (in JN you can do so from Settings > Jetpack Constants)
* You should be able to enable the Sharing module under Jetpack > Settings > Sharing
* Make sure the link to the Settings point to the settings page in WP-Admin

* Disable offline mode and enable user-less testing mode
* Connect only at a site level
* Make sure you're still able to enable/disable the module and the settings point to wp-admin

* Connect a user
* Make sure you're still able to enable/disable it
* Make sure the settings link points to calypso
